### PR TITLE
added more doc/comments to ellipsoidal chunk

### PR DESCRIPTION
**Pull request by bobmyhill**
This PR adds some comments to the ellipsoidal chunk, and rewords some of the error messages.

### CODE REVIEW

File: source/geometry_model/ellipsoidal_chunk.cc [hunk 1]
```diff
@@ -839,9 +851,11 @@ namespace aspect
                                    "two opposing points (SW and NE or NW and SE) a coordinate parallel ellipsoidal "
                                    "chunk geometry will be created. 2) by defining three points a non-coordinate "
                                    "parallel ellipsoidal chunk will be created. The points are defined in the input "
-                                   "file by longitude:latitude. It is also possible to define additional subdivisions of the "
-                                   "mesh in each direction. Faces of the model are defined as 0, west; 1,east; 2, south; 3, "
-                                   "north; 4, inner; 5, outer.\n\n"
+                                   "file by longitude:latitude. It is also possible to define additional subdivisions "
+                                   "of the mesh in each direction. The boundary of the domain joining two adjacent points "
+                                   "satisfies the expression [lon, lat](f) = [lon1*f + lon2*(1-f), lat1*f + lat2*(1-f)], "
+                                   "where f is a value between 0 and 1. Faces of the model are defined as "
```
Comment (MFraters) [inline source/geometry_model/ellipsoidal_chunk.cc]: To me this looks like a complicated way to say that there is a linear interpolation between the coordinates. Is that correct or am I misinterpreting it?

File: source/geometry_model/ellipsoidal_chunk.cc [hunk 2]
```diff
@@ -130,23 +137,28 @@ namespace aspect
...
-      const double b      = std::sqrt(R * R * (1 - eccentricity * eccentricity));
-      const double ep     = std::sqrt((R * R - b * b) / (b * b));
-      const double p      = std::sqrt(x(0) * x(0) + x(1) * x(1));
-      const double th     = std::atan2(R * x(2), b * p);
-      const double phi    = std::atan2(x(1), x(0));
-      const double theta  = std::atan2(x(2) + ep * ep * b * std::pow(std::sin(th),3),
-                                       (p - (eccentricity * eccentricity * R  * std::pow(std::cos(th),3))));
-      const double R_bar = R / (std::sqrt(1 - eccentricity * eccentricity * std::sin(theta) * std::sin(theta)));
-      const double R_plus_d = p / std::cos(theta);
+      const double R    = semi_major_axis_a; // semi-major axis
```
Comment (MFraters) [inline source/geometry_model/ellipsoidal_chunk.cc]: while you are at it, could you fix the indentation of the `=` at this line?

File: source/geometry_model/ellipsoidal_chunk.cc (hunk 1 above)
Comment (bobmyhill) [reply source/geometry_model/ellipsoidal_chunk.cc]: That's correct - I'll add a few more words to that effect, but will keep the math in unless you have strong objections.

File: source/geometry_model/ellipsoidal_chunk.cc (hunk 2 above)
Comment (bobmyhill) [reply source/geometry_model/ellipsoidal_chunk.cc]: Sure!

File: source/geometry_model/ellipsoidal_chunk.cc [hunk 3]
```diff
@@ -852,9 +852,10 @@ namespace aspect
                                    "chunk geometry will be created. 2) by defining three points a non-coordinate "
                                    "parallel ellipsoidal chunk will be created. The points are defined in the input "
                                    "file by longitude:latitude. It is also possible to define additional subdivisions "
-                                   "of the mesh in each direction. The boundary of the domain joining two adjacent points "
-                                   "satisfies the expression [lon, lat](f) = [lon1*f + lon2*(1-f), lat1*f + lat2*(1-f)], "
-                                   "where f is a value between 0 and 1. Faces of the model are defined as "
+                                   "of the mesh in each direction. The boundary of the domain is formed by linear "
+                                   "interpolation between adjacent points "
```
Comment (MFraters) [inline source/geometry_model/ellipsoidal_chunk.cc]: I though you wanted to say `linear interpolation in long-lat space` here?

**Pull request merged**